__tests__/api.test.js

```diff
@@ -58,4 +59,53 @@ describe("/api/getCategories", () => {
         });
       });
   });
+
+  it("error handling, expect 404 custom error if bad path", () => {
```

Comment:
This test is necessary, but it's a little bit confused. This should be a test for ANY unspecified endpoint. It will be the same for /carl /blorp or /categories/funny - therefore, we can't direct them to /api/categories because that won't always be the case!

Reply:
fixed :)